### DISCUSSION TITLE
Third Reality Blinds Can't Be Added to a Group

### DISCUSSION THREAD

**Discussion by majorsl** [Koenkk/zigbee2mqtt#18922] [Devices]
I've had this open for a bit now:  https://github.com/Koenkk/zigbee2mqtt/issues/18392

Just getting back to it to try again after a few updates and still no luck.  Does anyone else have these blinds and have added them to a group or would be willing to see if it is possible as a sanity check for me?

**Comment by darthrater78** [comment]
Also having the same issue.  I see this in the logs, running z2M 2.0, UZG01 coordinator with 20240909 firmware

z2m: Failed to add from group (ZCL command 0x282c02bfffe3f299/1 genGroups.add({"groupid":9,"groupname":""}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"reservedBits":0,"writeUndiv":false}) failed (Timeout - 22754 - 1 - 26 - 4 - 0 after 10000ms))

they operate fine by themseves

info 2025-01-09 18:46:39z2m:mqtt: MQTT publish: topic 'zigbee2mqtt/blind_office', payload '{"battery":100,"linkquality":91,"position":16,"state":"OPEN","update":{"installed_version":76,"latest_version":76,"state":"idle"},"update_available":false,"voltage":6200}'

**Comment by majorsl** [comment]
It has been awhile since I put this in, but from what I remember from either a forum or another Issue ticket, groups are not supported for either battery devices or cover devices. I kind of gave up on it and just group them in Home Assistant which works okay.
